TypeStyle/Generator/Transformers/StyleTransformers.swift

```diff
@@ -8,6 +8,10 @@ enum StyleTransformers {
     (groupName: "All Styles", transformers: [
       Transformer(name: "Monospace",
                   outputBase: "𝙰𝙱𝙲𝙳𝙴𝙵𝙶𝙷𝙸𝙹𝙺𝙻𝙼𝙽𝙾𝙿𝚀𝚁𝚂𝚃𝚄𝚅𝚆𝚇𝚈𝚉𝚊𝚋𝚌𝚍𝚎𝚏𝚐𝚑𝚒𝚓𝚔𝚕𝚖𝚗𝚘𝚙𝚚𝚛𝚜𝚝𝚞𝚟𝚠𝚡𝚢𝚣𝟷𝟸𝟹𝟺𝟻𝟼𝟽𝟾𝟿𝟶"),
+      
+      Transformer(name: "Fantasy",
+                  outputBase: "ꪖ᥇ᥴᦔꫀᠻᧁꫝ꠸꠹ᛕꪶꪑꪀꪮρꪇ᥅ᦓꪻꪊꪜ᭙᥊ꪗƺꪖ᥇ᥴᦔꫀᠻᧁꫝ꠸꠹ᛕꪶꪑꪀꪮρꪇ᥅ᦓꪻꪊꪜ᭙᥊ꪗƺ᧒ᒿᗱᔰƼᦆᒣᲖၦᦲ"),
+      
```

Comment:
For consistency, can you remove the extra lines above and below the Fantasy Transformer?

```suggestion
      Transformer(name: "Fantasy",
                  outputBase: "ꪖ᥇ᥴᦔꫀᠻᧁꫝ꠸꠹ᛕꪶꪑꪀꪮρꪇ᥅ᦓꪻꪊꪜ᭙᥊ꪗƺꪖ᥇ᥴᦔꫀᠻᧁꫝ꠸꠹ᛕꪶꪑꪀꪮρꪇ᥅ᦓꪻꪊꪜ᭙᥊ꪗƺ᧒ᒿᗱᔰƼᦆᒣᲖၦᦲ"),
```